rt4914/KlinicCon-Frontend
# Fixes #26 feature request custom sociallink component

## What type of PR is this? (check all applicable)


## Description

Social link component that will have facebook, instagram & twitter as icons which is reusable component like providing width and height as props for adjusting the width, height of the SVG Icon

## Related Tickets & Documents

- Related Issue #26
- Closes #26 

## QA Instructions, Screenshots, Recordings

Tested by including social link component in App.jsx, below is what we see

![CleanShot 2024-09-19 at 12 49 42](https://github.com/user-attachments/assets/e727bd91-5254-41d8-bce2-a19619c94de4)

While hovering over the icon, below is what we see (background of the icon would be changed)

![CleanShot 2024-09-19 at 12 50 44](https://github.com/user-attachments/assets/6fbd1302-4697-4740-bf0a-cb35bafb5a03)
